app/ApiObjects/Topic.php

```diff
@@ -25,6 +25,7 @@ public function toArray()
             'title' => $this->topic->title,
             'description' => $this->topic->description,
             'votes' => $this->topic->votes()->count(),
+            'url' => $this->topic->url,
```

Comment:
Hm, I was expecting #7 to reference a URL on an episode, not on a topic. I don't know if I'm *against* the idea of allowing a URL on a topic.. but just for what it's worth, that's a separate issue than #7.